Sources/FingerprintJSTests/Tests/IdentifierHarvesterTests.swift

```diff
@@ -146,4 +155,60 @@ final class IdentifierHarvesterTests: XCTestCase {
         let itemLabels = itemsTree.children?.map(\.label) ?? []
         XCTAssertTrue(itemLabels.isEmpty)
     }
+
+    func
+        test_givenBgIdentifierMissingLegacyIdentifierPresent_whenVendorIdentifier_thenMigratesOldIdentifierToNewStorage()
```

Comment:
Once again, let's maybe try to avoid calling it a background identifier for the reason mentioned in the previous comment.
```suggestion
        test_givenIdentifierMissingLegacyIdentifierPresent_whenVendorIdentifier_thenMigratesLegacyIdentifierToNewStorage()
```